### PR TITLE
Feature/parse model params from config file

### PR DESCRIPTION
**Pull request by dchvs99**

### CODE REVIEW

**Comment by dchvs99** [comment]
This PR syncs the Demo to work with the Edge AI's SDK 8.

File: config.yaml [hunk 1]
```diff
@@ -1,21 +1,27 @@
+model_params:
+  disp_width: '320'
```
Comment (michaelgruner) [inline config.yaml]: You should make these integers (remove quotes)

File: rr/config/app_validator.py [hunk 2]
```diff
@@ -243,6 +243,48 @@ def validate_triggers(cfg, actions, filters):
...
+        list,
+        "Model parameters field not found",
+        "Found model parameters field, but is is not a list")
+    model_params = cfg['model_params']
+
+    for model_param in model_params:
+        validate_objects(
+            model_param,
+            'disp_width',
+            str,
```
Comment (michaelgruner) [inline rr/config/app_validator.py]: Validates these to be integers, not strings.

**Review by michaelgruner — APPROVED** [review]
Approved with changes

**Pull request merged**